webapp/src/ts/providers/translation-loader.provider.ts

```diff
@@ -14,7 +14,7 @@ export class TranslationLoaderProvider implements TranslateLoader {
 
   getTranslation(locale) {
     if (!locale) {
-      return;
+      return undefined as any;
```

Comment:
This is probably the worst change I had to make since it is allowing the function to return a value (`undefined`) that is explicitly against the return type for the interface (`Observable<any>`).  This function is never supposed to return null/undefined.  That being said, the `locale` parameter is also not technically allowed to be null/undefined so I don't think we could ever hit this code in prod....